boards/nxp/imx8mm_evk/imx8mm_evk_mimx8mm6_a53_ramconsole.dts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_mimx8mm_a53.dtsi>
+#include "imx8mm_evk-pinctrl.dtsi"
+
+/ {
+	model = "NXP i.MX8MM A53";
+	compatible = "fsl,mimx8mm";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,ram-console = &ram_console;
```

Comment:
Any particular reason this wouldn't be a snippet instead?

Reply:
This is suggested by Carlo, refer to https://github.com/zephyrproject-rtos/zephyr/pull/68452#discussion_r1477925702

Reply:
If this is a snippet then the existing board targets (including variants) can be used as they are, and optionally switched to RAM console by doing e.g. `-DSNIPPET=imx8mm_ram_console` or similar. Thoughts @gmarull @tejlmand ?

Reply:
I misunderstood your previous comment, I didn't know the snippet in Zephyr.
It's a good idea to use the snippet. Thanks for the suggestion!